packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -349,8 +345,7 @@ describe.each([
         await config.api.viewV2.create(newView, {
           status: 400,
           body: {
-            message:
-              'Field "name" cannot be readonly as it is a required field',
+            message: 'You can\'t make read only the required field "name"',
```

Comment:
This reads a little strange, I think: "You can't make field "name" readonly because it is a required field." would be better.